portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.data;
+
+public class Comment{
+    //personal data
+    private String firstName, lastName, email, phone;
+    //job related data: optional
+    private boolean jobRelated=false;
+    private String jobTitle="";
+    //message 
+    private String message;
+
+    public Comment(String firstName, String lastName, String email, String phone, String message){
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.email = email;
+        this.phone = phone;
+        this.message = message;
+    }
+
+    public void setJobRelated(){
+        this.jobRelated = true;
+    }
+
+    public void addJobTitle(String jobTitle){
+        this.jobTitle = jobTitle;
+    }
```

Comment:
We could also make these part of the constructor, in order to have all fields final. 
Also, when the number of constructor parameters starts to grow, we can start thinking about using the builder pattern. 

http://go/ej3e-2

Reply:
I removed the jobRelated field, and thus I have only one optional field left, so I decided to keep the addJobTitle() function